src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -662,6 +669,32 @@ public function getOfficerships()
         return $this->officerships;
     }
 
+    /**
+     * Get the User's radio time, calculated from timeslot signins
+     */
+    public function getRadioTime()
+    {
+        if (!$this->radiotime) {
+            $this->radiotime = self::$db->fetchColumn(
```

Comment:
No error checking? Does fetchColumn yeet errors or just return false or something like that?